src/app.js

```diff
@@ -0,0 +1,10 @@
+require('dotenv').config();
+
+const express = require('express');
+const app = express();
+
+app.get('/', (req, res) => {
+  res.send('Hello World!');
+});
```

Comment:
Can we delete this part if it not used ?

Reply:
hummm, we are using it for the first test, maybe we can remove it after ?